### DISCUSSION TITLE
Made GUI that works on windows, mac and linux that uses zim files that makes scraping using Zim files so much easier

### DISCUSSION THREAD

**Discussion by Nicholas55555** [kiwix/.github#11] [Show and tell]
Check it out on my github: https://github.com/Nicholas55555/Zim-Assistant-Gui

![image](https://github.com/user-attachments/assets/ea3c23fd-f620-4ddc-b738-7158d229e3e5)

I was annoyed by the lack of support for openzim for windows and the difficulty/complication of creating and extracting text from Zim files for scraping so I decided to make a GUI for it myself.

**Comment by Nicholas55555** [comment]
Feel free to follow me on GitHub and star my project. Also I would greatly appreciate it if you let me know of any issues or things I should add.

**Comment by rgaudin** [comment]
Good to know ; thank you!

**Comment by catmaps** [comment]
Good idea here! Yes, please proceed with it! I wish for easy way to reach into a zim file and fix annoying format or text in a given article and then let it repack right back into zim file. Existing manipulation tools are powerful but clumsy; such as:  https://gitlab.com/Afrikalan/zim-tools/-/tree/master/zim-manipulation
https://pypi.org/project/python-zim/
